frontend/liferay-theme/src/main/webapp/templates/portal_normal.ftl

```diff
@@ -62,7 +62,12 @@
 	<header id="banner" role="banner">
 		<div id="heading" class="container">
             <#if is_signed_in>
-                <#assign expandoAttribute = liferay_user.getExpandoBridge().getAttribute("BannerMessage") />
+                <#if liferay_user.getExpandoBridge().getAttribute("BannerMessage")??>
+                    <#assign expandoAttribute = liferay_user.getExpandoBridge().getAttribute("BannerMessage") />
+                <#else>
+                    ${liferay_user.getExpandoBridge().addAttribute("BannerMessage")}
+                    <#assign expandoAttribute = liferay_user.getExpandoBridge().getAttribute("BannerMessage") />
+                </#if>
                 <#if expandoAttribute?has_content>
```

Comment:
I would suggest just to make the change at this line, rather than adding `if-else` clause.. something like this...
`<#if liferay_user.getExpandoBridge().getAttribute("BannerMessage")?has_content>`